### ISSUE TITLE
Hasql 0.X

### ISSUE THREAD

**Issue by nikolaipaul** [nikita-volkov/hasql#108]
is your work on the experimental version on hold for now?

**Comment by nikita-volkov** [comment]
Yes, unfortunately. I do plan to get back to it, once I get some spare time.

**Comment by paolino** [comment]
is there any progress to get to  streaming results ?

**Comment by nikita-volkov** [comment]
Unfortunately I haven't worked on that in the past year. It's still on my list though.

What kind of sponsorship are we talking about?

**Comment by danstn** [comment]
Hey @nikita-volkov! At the moment we're using Opaleye and Hasql looks like the only viable option to switch to. Just curious if there are any plans to resume this? Really interested in the streaming results part + finally ditching libpq would be amazing :raised_hands:

**Comment by nikita-volkov** [comment]
I've been coming back to work on this from time to time. I do have plans to finish with it, but I just can't say when
